src/app/containers/ATIAnalytics/atiUrl/index.test.js

```diff
@@ -113,3 +113,23 @@ describe('getThingAttributes', () => {
     expectedValues.forEach(value => expect(queryParamsArray).toContain(value));
   });
 });
+
+describe('buildATIEventTrackUrl', () => {
+  it('should return the right url', () => {
+    process.env.SIMORGH_ATI_BASE_URL = 'http://foobar.com?';
+    expect(
+      buildATIEventTrackUrl({
+        pageIdentifier: 'pageIdentifier',
+        service: 'service',
+        platform: 'platform',
+        statsDestination: 'statsDestination',
+        element: 'element',
+        component: 'component',
+        label: 'label',
+        type: 'type',
+      }),
+    ).toEqual(
```

Comment:
I think you should make the value more readable like

```js
`http://foobar.com?${[
  's=getDestination',
  'p=pageIdentifier',
  'r=getScreenInfo',
  're=getBrowserViewPort',
  'hl=getCurrentTime',
  'lng=getDeviceLanguage',
  'ati=PUB-[service-component]-[=type]-[label]-[PAR=container-component::name~CHD=brand-top]-[]-[]-[]-[/]',
].join('&')}`;
```

now it's in a more readable format, are you sure it's correct? The query param values don't look right to me.

Reply:
Fixed the formatting.

See explanation in simorgh/src/app/lib/analyticsUtils/index.js from line 180